### PR TITLE
Create tab navigation

### PR DESCRIPTION
**Pull request by victor-dantas**
Create new two components for bh called Bh-tabs and Bh-tab-item

### CODE REVIEW

File: bh/documentation/src/App.vue [hunk 1]
```diff
@@ -175,6 +175,15 @@
           </div>
         </bh-card>
       </section>
+      <section class="bh-grid__item">
+        <bh-card>
+          <div class="bh-grid">
+            <div class="bh-grid__item bh-grid__item--nested">
+              <TabNavigation />
```
Comment (douglasppaz) [inline bh/documentation/src/App.vue]: `<TabNavigation />` -> `<tab-navigation />`

File: bh/documentation/src/tab-navigation/TabNavigation.vue [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    <div class="bh-grid__item">
+      <api :data="api" />
+    </div>
+  </div>
+</template>
+
+<script>
+import Example from '@/Example';
+import Api from '@/Api';
+import TabNavigationExample from './examples/TabNavigationExample';
```
Comment (douglasppaz) [inline bh/documentation/src/tab-navigation/TabNavigation.vue]: Rename this file `TabNavigationExample` to `TabNavigation`, but keep TabNavigationExample as component name

File: bh/documentation/src/tab-navigation/examples/TabNavigationExample.vue [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<template>
+  <div>
+    <bh-tabs v-model="activeTab">
+      <bh-tab-item label="Food">
+        Nunc nec velit nec libero vestibulum eleifend.
```
Comment (douglasppaz) [inline bh/documentation/src/tab-navigation/examples/TabNavigationExample.vue]: Put a button with tooltip inside to test overflow.

File: bh/src/components/BhTabItem.vue [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import '@scss/variables.scss';
+
+
+.slide-next-enter-active,
```
Comment (douglasppaz) [inline bh/src/components/BhTabItem.vue]: create a sass hierarchy.

```
.slide {
  &-next {...}
}
```

File: bh/src/components/BhTabs.spec.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  beforeEach(() => {
+    wrapper = shallowMount(BhTabs, {
+      localVue,
+    });
+  });
+
+  test('renders correctly', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+});
```
Comment (douglasppaz) [inline bh/src/components/BhTabs.spec.js]: Create more tests.. with slots, transactions.

File: bh/documentation/src/tab-navigation/examples/TabNavigationExample.vue (hunk 3 above)
Comment (victor-dantas) [reply bh/documentation/src/tab-navigation/examples/TabNavigationExample.vue]: On tab-item called 'Music' there is a button with tooltip.

**Pull request merged**